### DISCUSSION TITLE
I can't include previously excluded/factory reset device

### DISCUSSION THREAD

**Discussion by Roguyt** [zwave-js/zwave-js#7048] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening? / What did you expect to happen instead?
After having some trouble with a Shelly Qubino Wave device, I had to exclude it to reinclude it.
However it has been impossible since the driver keeps sending 
```
Dropping message because it could not be deserialized: The command class unkno
                                  wn (0xf206) is not implemented (ZW0303)
```

every time an inclusion process starts.

I've tried pretty much everything:
- Restarted fully the host
- Unplugged the stick for a while
- Excluded another device
- Healed the network

What's really weird is that the device seems to believe it's included since it displays the green OK led.

  1. Exclude a device
  2. Try to reinclude it
  3. See error in the log

Anything else we should know?
No

### Software versions

zwave-js-ui: 9.16.2.37dfcd3
zwave-js: 13.0.2
home id: 4060505610
home hex: 0xf206660a

homeassistant and zwavejsui in docker with mqtt 2024.8.0

### Device information

Silicon Labs | USB Controller | 700/800 Series | Controller
Shelly Europe Ltd. | Wave 1PM | QNSW-001P16 | Shelly Corridor x 2

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_current.log](https://github.com/user-attachments/files/16388113/zwavejs_current.log)

**Comment by AlCalzone** [comment]
Pretty sure this is the controller firmware doing something strange. The frames that are received start with your home ID `0xf206...`, but the controller should be extracting the actual payload from there.

The only option I see is upgrading the firmware, but 7.20 and 7.21 are pretty much broken at this point, and 7.22 is 800 series only.

**Reply by Roguyt** [reply]
Could downgrading to 7.18 be a possibility before reflashing 7.19 ?

**Reply by AlCalzone** [reply]
Usually Z-Wave controllers cannot be downgraded, unless you have a bootloader where that check is removed.

**Comment by Roguyt** [comment]
I've tried resetting the stick inside Simplicity Studio, and managed to pair one of the two devices. But I can't pair both of them at the same time.

So besides waiting for a new version from SiliconLabs I have no alternative :(

**Reply by Roguyt** [reply, marked as answer]
Updated to 7.19.5. included both devices without any issues first try. I guess it was indeed a weird firmware behavior. Thanks @AlCalzone !
